libr/core/cmd_write.c

```diff
@@ -740,7 +740,7 @@ static bool cmd_wfs(RCore *core, const char *input) {
 			done += rc;
 		}
 		r_socket_free (c);
-		if (r_io_write_at (core->io, core->offset, buf, done)) {
+		if (r_core_write_at (core, core->offset, buf, done)) {
```

Comment:
I do t think @condret will be happy with this change

Reply:
r_core_write_at should die. so dont use it. maybe adding hooks to rio is the way to go here